### ISSUE TITLE
Reading sample puzzle

### ISSUE THREAD

**Issue by AmitPatil999** [Borroot/akari#1]
What does capital B in the sample puzzle indicate? a Black cell or Bulb?
'e0a1c2cBqBcBc3a2e'

**Comment by Borroot** [comment]
B indicates a wall (black cell)!

**Comment by AmitPatil999** [comment]
Thanks for your prompt response. Where can I find akari puzzles of various
other grid sizes apart from what you have provided in the "internet"
folder? Also can you provide solutions to all the puzzles from that folder.
I don't know how to execute Python.

**Comment by Borroot** [comment]
You can generate puzzles of any size using `generator.py`, note that these can be extremely difficult. You can run `z3solver.py` on any given puzzle to get its solution. May I ask what is your use case so I understand your problem better?

**Comment by AmitPatil999** [comment]
I have developed a web app for the puzzle. No requirement for generating the new puzzles, as the puzzles I got are more than enough. (It's a daily game and I wanted 365 puzzles in each grid size and difficulty, which I got.) I just wanted to check if the user solved answer is correct or not. For that I could match it with the pre-recorded answer if available. That's why I asked. 
But I solved the problem by writing the logic to check the answer myself. 
Here's the app link if you're interested. = https://marathigames.in/LightUp/lightup.html
It's in Indian language, Marathi.
